corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-webapp/menu.scss

```diff
@@ -12,3 +12,15 @@
 #menu-region .module-menu-container {
   padding-bottom: 6px;
 }
+
+#persistent-menu-container {
+  background-color: rgb(0 0 0 / 25%);
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1000;
+
+  #persistent-menu-content {
+    background-color: beige;
+  }
+}
```

Comment:
This was all temporary stuff in helping me build this out - it gets replaced later